tests/testthat/test-upkeep.R

```diff
@@ -1,56 +1,49 @@
 test_that("tidy upkeep bullets don't change accidentally", {
-  withr::local_options(
-    usethis.description = list(
-      "Authors@R" = utils::person(
-        "Jane", "Doe",
-        email = "jane@rstudio.com",
-        role = c("aut", "cre")
-      ),
-      License = "MIT + file LICENSE"
-    )
-  )
   create_local_package()
+  use_mit_license()
 
-  expect_snapshot(
-    writeLines(tidy_upkeep_checklist(posit_pkg = TRUE, posit_person_ok = FALSE)),
-    transform = scrub_checklist_footer
+  local_mocked_bindings(
```

Comment:
I probably should've done this in a separate PR, but using mocking for everything makes the tests quite a bit simpler.